README.md

```diff
@@ -1,92 +1,72 @@
-# Noora ⭐️
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+# Noora Design System
 
-Command Line Interfaces (CLIs), though graphically limited due to terminal capabilities, **can still benefit significantly from well-designed and consistent aesthetics across various commands**. This is a role traditionally filled by design systems in Graphical User Interfaces (GUIs), but it remains largely unexplored in the context of terminals.
+Noora is Tuist's comprehensive design system that provides consistent UI components and patterns across all platforms and environments.
 
-Tuist's terminal experiences, for a long time, suffered from the lack of a design system, leading to issues like inconsistent spacing, information overload, and difficult-to-parse outputs. This situation motivated us to create SwiftTerminal, **a design system specifically for Swift-powered CLIs.**
+## Packages
 
-Noora offers a set of aesthetically pleasing, customizable, and uniform design system for crafting terminal experiences. It encapsulates our extensive experience in crafting the Tuist CLI. We are delighted to share it with the Swift community, aiming to make building terminal experiences an enjoyable process.
+This monorepo contains multiple packages that implement Noora's design principles:
 
-> [!NOTE]
-> The project is currently in an early stage of development. Our plan is to create a basic set of components and then iterate on them based on the feedback we receive from the community.
+### 📱 CLI Package (`packages/cli/`)
 
-## Usage
+A Swift package providing terminal UI components for building beautiful command-line interfaces.
 
-Add `Noora` as a dependency of your project:
+### 🌐 Web Package (`packages/web/`) _(Coming Soon)_
 
-```swift
-import PackageDescription
+A package to build interactive user interfaces for the web using Phoenix LiveView.
 
-let package = Package(
-    name: "Noora",
-    platforms: [.macOS("12.0")],
-    products: [
-     /** Your products **/
-    ],
-    dependencies: [
-      .package(url: "https://github.com/tuist/Noora", .upToNextMajor(from: "0.15.0")),
-    ],
-    targets: [
-      /** Your targets **/
-    ]
-)
-```
+## Development
 
-Then you can start using Noora.
-You need to create an instance of `Noora` first, and then every component available is represented as a function in the `Noora` instance.
+### Prerequisites
 
-```swift
-Noora().yesOrNoChoicePrompt(
-  title: "Authentication",
-  question: "Would you like to authenticate?",
-  defaultAnswer: true,
-  description: "Authentication is required to use some CLI features."
-)
-```
+- [mise](https://mise.jdx.dev/) for tool management
+- Swift 6.0+ (for CLI package)
+- Node.js 22+ (for web package)
```

Comment:
Since this information is already in the `mise.toml`. Should we just point people to running `mise install` or checking the dependencies in that file?